README.md

```diff
@@ -354,6 +354,22 @@ For more information about the Storage tool see https://graphdb.ontotext.com/doc
 By default, GraphDB's Helm chart comes with a default Ingress and also Kong for more flexibility in configuring instances paths.
 Both the Ingress and Kong can be disabled by switching `kong.enabled` and `ingress.enabled`.
 
+### Cloud deployments specifics
+
+Some cloud kubernetes clusters have some specifics that should be noted. Here are some useful tips on some cloud K8s clusters:
+
+##### Google cloud
+
+In Google's k8s cluster services, the root directory is not writable. By default GraphDB's chart uses `/data` directory to store instances data.
```

Comment:
maybe we should do this in the chart by default and avoid this note something similar is done in bitnami's postgresql chart